Fix/listings help link

phattya opened the pull request:
# Pull Request Template

## Issue Overview

This PR addresses #issue

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description
Add href for link to the help center housing-help page on the Listings Need Help card.

## How Can This Be Tested/Reviewed?

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

ifranch reviewed (changes requested):
Looks like a lot more commits got added to this PR, can you rebase and push just the one commit?

phattya commented:
> Looks like a lot more commits got added to this PR, can you rebase and push just the one commit?

@ifranch yeah...I didn't realize it until after I added y'all as reviewers...so removed you & was trying to fix before reassigning, but you were too quick! Thanks! Fixed now.

ifranch commented on sites/public/src/components/listings/ListingsList.tsx:
```diff
@@ -48,7 +48,9 @@ const ListingsList = (props: ListingsListProps) => {
           className="is-normal-secondary-lighter"
         >
-          <Button className="is-secondary">{t("t.helpCenter")}</Button>
+          <LinkButton href="help/housing-help" className="is-secondary">
```
I think this should start with / so /help/housing-help

phattya replied:
Geesh. How did I miss that. Tho it did work when I tested it locally. Fixed now! Thanks!
